plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialAttached.java

```diff
@@ -53,6 +55,7 @@ public static void register() {
         // For a lantern, this returns whether it is hanging from the ceiling.
         // For a gate, this returns whether it is lowered to attach to a wall block.
         // For a mangrove_propagule, this returns whether it is hanging from the block above it.
+        // For a tripwire, this returns whether a tripwire hook or string forms a complete tripwire circuit and is ready to trigger.

```

Comment:
You've also implicitly added support for hanging signs, the behavior of which should be documented